python/lsst/meas/algorithms/psfDeterminer.py

```diff
@@ -34,22 +34,51 @@ class BasePsfDeterminerConfig(pexConfig.Config):
 
     This is fairly sparse; more fields can be moved here once it is clear they are universal.
     """
+    stampSize = pexConfig.Field(
```

Comment:
I see you used the new type annotation syntax elsewhere on this PR, but not here.  Any reason why?  (I don't have a preference, except for consistency.)

Reply:
I did that in one file, where there were only two config fields. I'm happy to change it in the files I'm touching.